src/pybind/mgr/cephadm/templates/services/grafana/ceph-dashboard.yml.j2

```diff
@@ -5,6 +5,9 @@ deleteDatasources:
     orgId: 1
 {% endfor %}
 
+  - name: 'Loki'
+    orgId: 1
```

Comment:
What if I don't want to deploy the centralized logging?

Reply:
This is just for adding the Loki datasource in grafana. We need to create loki and promtail services explicitly to access this datasource